<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>1061</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="body" native="true">
      <layout class="QVBoxLayout" name="bodyLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="plotContainer">
         <item>
          <widget class="QWidget" name="xPlotContainer" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QWidget" name="shotDetailBar" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="shotLabel">
                 <property name="maximumSize">
                  <size>
                   <width>35</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Shot</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="shotNumberEdit">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Insert a shot number and press enter to load shot data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="statDurationLabel">
                 <property name="maximumSize">
                  <size>
                   <width>55</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Duration:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="dynDurationLabel">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The duration of the shot&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnELMplot">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Create ELM region plot</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QComboBox" name="switchxPlot">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select which quantity to show as a function of delta s&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <item>
                 <property name="text">
                  <string>Density</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Temperature</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Saturation current density</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QWidget" name="xPlot" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="xPlotLayout"/>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="xPlotControls" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="lblStatTime">
                 <property name="text">
                  <string>Time</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="xTimeSlider">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Browse through the temporal evolution of the shot&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="xTimeEdit">
                 <property name="maximumSize">
                  <size>
                   <width>60</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Insert a specific time that you would like the plot to show&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblTWidth">
                 <property name="text">
                  <string>Window width</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinTWidth"/>
               </item>
               <item>
                <widget class="QLabel" name="lblRealTWidth">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="probeTable">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>500</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>xPlot</zorder>
           <zorder>xPlotControls</zorder>
           <zorder>shotDetailBar</zorder>
           <zorder>probeTable</zorder>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="tPlotContainer" native="true">
           <layout class="QVBoxLayout" name="tPlotContainerLayout">
            <item>
             <widget class="QWidget" name="TPlot" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="TPlotLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="nPlot" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <layout class="QVBoxLayout" name="nPlotLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="jPlot" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="jPlotLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="tPlotControls" native="true">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="tPlotCtrlLayout">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>9</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="buttonContainer" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>200</height>
                  </size>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="sizeConstraint">
                   <enum>QLayout::SetMaximumSize</enum>
                  </property>
                  <item>
                   <widget class="QPushButton" name="btnPan">
                    <property name="text">
                     <string>Pan</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnZoom">
                    <property name="text">
                     <string>Zoom</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnReset">
                    <property name="text">
                     <string>Reset</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1234</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave_as">
     <property name="title">
      <string>Save as</string>
     </property>
     <addaction name="action_png"/>
    </widget>
    <addaction name="menuSave_as"/>
    <addaction name="actionExport"/>
    <addaction name="actionImport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuSpatial_Plot">
     <property name="title">
      <string>Spatial Plot</string>
     </property>
     <addaction name="menuIgnoreNaNsSpatial"/>
    </widget>
    <addaction name="actionToggle_plot"/>
    <addaction name="actionMaximize_plot"/>
    <addaction name="menuFixxPlotyLim"/>
    <addaction name="menuSpatial_Plot"/>
   </widget>
   <widget class="QMenu" name="menuShot">
    <property name="title">
     <string>Shot</string>
    </property>
    <widget class="QMenu" name="menuDiagnostic">
     <property name="title">
      <string>Diagnostic</string>
     </property>
     <addaction name="actionLSD"/>
     <addaction name="actionLSF"/>
    </widget>
    <widget class="QMenu" name="menuProbe_array">
     <property name="title">
      <string>Probe array</string>
     </property>
     <addaction name="actionLower_outer_array"/>
    </widget>
    <addaction name="actionLoad_new_shot"/>
    <addaction name="menuDiagnostic"/>
    <addaction name="menuProbe_array"/>
   </widget>
   <widget class="QMenu" name="menuStatistis">
    <property name="title">
     <string>Statistics</string>
    </property>
    <widget class="QMenu" name="menuFiltering">
     <property name="title">
      <string>Filtering</string>
     </property>
     <widget class="QMenu" name="menuELMs">
      <property name="title">
       <string>ELMs</string>
      </property>
      <addaction name="actionSet_threshold_value"/>
     </widget>
     <addaction name="menuELMs"/>
     <addaction name="actionZeroes"/>
     <addaction name="actionCusto_m_filter"/>
     <addaction name="actionFilter_NaNs"/>
    </widget>
    <widget class="QMenu" name="menuAvg">
     <property name="title">
      <string>Averaging</string>
     </property>
     <addaction name="menuAvgSpatial"/>
     <addaction name="menuAvgTemporal"/>
    </widget>
    <addaction name="actionRedo_Statistics"/>
    <addaction name="separator"/>
    <addaction name="menuAvg"/>
    <addaction name="menuFiltering"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuGraphs">
     <property name="title">
      <string>Graphs</string>
     </property>
     <addaction name="actionColoring_2"/>
     <addaction name="actionStyle"/>
    </widget>
    <addaction name="menuGraphs"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuShot"/>
   <addaction name="menuStatistis"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="action_png">
   <property name="text">
    <string>*.png</string>
   </property>
  </action>
  <action name="actionToggle_plot">
   <property name="text">
    <string>Toggle plot</string>
   </property>
  </action>
  <action name="actionMaximize_plot">
   <property name="text">
    <string>Maximize plot</string>
   </property>
  </action>
  <action name="actionLoad_new_shot">
   <property name="text">
    <string>Load new shot</string>
   </property>
  </action>
  <action name="actionLSD">
   <property name="text">
    <string>LSD</string>
   </property>
  </action>
  <action name="actionLSF">
   <property name="text">
    <string>LSF</string>
   </property>
  </action>
  <action name="actionLower_outer_array">
   <property name="text">
    <string>Lower outer array</string>
   </property>
  </action>
  <action name="actionSet_threshold_value">
   <property name="text">
    <string>Set threshold value</string>
   </property>
  </action>
  <action name="actionZeroes">
   <property name="text">
    <string>Filter Zeroes</string>
   </property>
  </action>
  <action name="actionCusto_m_filter">
   <property name="text">
    <string>Custom filter</string>
   </property>
  </action>
  <action name="actionRedo_Statistics">
   <property name="text">
    <string>Redo Statistics</string>
   </property>
  </action>
  <action name="actionColoring_2">
   <property name="text">
    <string>Coloring</string>
   </property>
  </action>
  <action name="actionStyle">
   <property name="text">
    <string>Style</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFilter_NaNs">
   <property name="text">
    <string>Filter NaNs</string>
   </property>
  </action>
  <action name="actionNumber_of_points">
   <property name="text">
    <string>Number of points</string>
   </property>
  </action>
  <action name="menuAvgSpatial">
   <property name="text">
    <string>Spatial plots</string>
   </property>
  </action>
  <action name="menuAvgTemporal">
   <property name="text">
    <string>Temporal plots</string>
   </property>
  </action>
  <action name="menuFixxPlotyLim">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fix spatial plot height</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="menuIgnoreNaNsSpatial">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ignore NaNs</string>
   </property>
   <property name="toolTip">
    <string>Ignoring NaNs will ensure plotting of points even if one of the data points they represent is NaN</string>
   </property>
  </action>
  <action name="actionStuuff">
   <property name="text">
    <string>stuuff</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
